js/modules/k6/metrics/metrics.go

```diff
@@ -75,6 +77,21 @@ func (mi *ModuleInstance) newMetric(call goja.ConstructorCall, t stats.MetricTyp
 	return v.ToObject(rt), nil
 }
 
+const warnMessageValueMaxSize = 100
+
+func limitValue(v string) string {
+	if len([]rune(v)) < warnMessageValueMaxSize {
+		return v
+	}
+	vRunes := []rune(v)
```

Comment:
you can move this to the top of the function and then check `len(vRunes)` to avoid converting it twice

Reply:
I was thinking that `len([]rune(v))` might skip the making of the slice ... but looking at this again I kind of doubt it :facepalm: so I am gonna do it.